### ISSUE TITLE
Warning in console when importing `logtail-js` into my project

### ISSUE THREAD

**Issue by msafi** [logtail/logtail-js#14]
Hi. Thank you for your work on this library!

I have a [Next.js project](https://github.com/hollowverse/hollowverse) where I'm using this library. But when I include it, my console gets spammed with the following message

```
./node_modules/@logtail/node/dist/es6/context.js
Critical dependency: require function is used in a way in which dependencies cannot be statically extracted
```

<img width="986" alt="image" src="https://user-images.githubusercontent.com/4027731/171515674-2d77e0cf-74c5-4631-a2fa-ebc907f1ccf2.png">

Thanks for looking into this!

**Comment by moroshko** [comment]
Same here. Any update on this? @simonrozsival

**Comment by curusarn** [comment]
Thank you for reporting this.
We are tracking this issue internally. We will make sure to keep you posted.

I'm closing this issue as a duplicate of #20

**Comment by curusarn** [comment]
Hi everyone,

I have just released [v0.3.0](https://github.com/logtail/logtail-js/releases/tag/v0.3.0) which shouldn't give you warnings anymore. 
Update to `v0.3.0` and let me know if the issue persists.

You can always message us at [hello@betterstack.com](mailto:hello@betterstack.com). We're happy to help. 🙏
